### PR TITLE
Add includes '\i' functionality

### PR DESCRIPTION
**Pull request by mikekaminsky**
I'm sort of new to Python so sorry if this isn't great. It appears to work, but I wasn't sure how to get the test db set up so I didn't...run any. Also I sort of hacked the use of the @special_command decorator, but since it seems to work as I expected it, I left it.

I'm new to open source stuff, so I'd be happy to rework and resubmit if you give me any feedback.

### CODE REVIEW

File: pgcli/packages/pgspecial/iocommands.py [hunk 1]
```diff
@@ -68,6 +68,30 @@ def open_external_editor(filename=None, sql=''):
 
     return (query, message)
 
+@special_command('\\i', '\\i file', 'List or execute named queries.')
```
Comment (amjith) [inline pgcli/packages/pgspecial/iocommands.py]: The documentation for this should say 'Execute commands from file'.

File: pgcli/packages/pgspecial/iocommands.py [hunk 2]
```diff
@@ -68,6 +68,30 @@ def open_external_editor(filename=None, sql=''):
...
+def execute_from_file(cur, pattern, **_):
+    if pattern != '':
```
Comment (amjith) [inline pgcli/packages/pgspecial/iocommands.py]: In python empty strings are falsy. So you can change this `if` condition as follows:

```
if pattern:
```

File: pgcli/packages/pgspecial/iocommands.py [hunk 3]
```diff
@@ -68,6 +68,30 @@ def open_external_editor(filename=None, sql=''):
...
+        try:
+            query = read_from_file(pattern)
+        except IOError, e:
```
Comment (amjith) [inline pgcli/packages/pgspecial/iocommands.py]: This should say 

```
except IOError as e:
```

Otherwise the tests will fail in Python 3.

**Comment by amjith** [comment]
This PR is a great start. I've left some comments inline to fix some obvious shortcomings.

Once this PR is merged, I might extend this feature to do more. Such as handle multiple queries. 

But otherwise it's a good PR. As son as you address the comments, I'll merge it in.

Thank you for taking the time to implement the feature.

**Comment by mikekaminsky** [comment]
Thanks for the comments! I've added the fixes and resubmitted.

**Comment by amjith** [comment]
Looks good. Thank you for fixing the issues. 

Cheers!

:shaved_ice:

**Pull request merged**